lib/interfaces/include/VCFInterface.h

```diff
@@ -0,0 +1,39 @@
+#ifndef __VCFINTERFACE_H__
```

Comment:
Why do we have PedalsStampedData as a wrapper for PedalsSystemData? Where is TimestampedData defined? Why must all data be timestamped?